.gitignore

```diff
@@ -496,12 +496,6 @@ devenv.local.nix
 
 *.actual
 
-# Python
-# Byte-compiled / optimized / DLL files
-__pycache__/
-*.py[cod]
-*$py.class
-
```

Comment:
The flowing line in the `.gitignore` file are also python related and should be removed